packages/thirdweb/src/wallets/in-app/native/helpers/wallet/sharded-wallet.ts

```diff
@@ -67,6 +70,7 @@ export class ShardedWallet implements IWebWallet {
     return getExistingUserAccount({
       client: this.client,
       storage: this.storage,
+      storedToken: this.storedToken,
```

Comment:
this feels scary. if you have it here in memory, can it ever be different form the one in local storage?